creator-node/test/tracks.test.js

```diff
@@ -489,7 +489,7 @@ describe('test Tracks with real IPFS', function () {
     // check that the generated transcoded track is the same as the transcoded track in /tests
     const transcodedTrackAssetPath = path.join(__dirname, 'testTranscoded320Track.mp3')
     const transcodedTrackAssetBuf = fs.readFileSync(transcodedTrackAssetPath)
-    const transcodedTrackPath = path.join(storagePath, resp.body.data.transcodedTrackCID)
+    const transcodedTrackPath = path.join(storagePath, resp.body.transcodedTrackCID)
```

Comment:
is this because we don't sign the json response so it doesn't copy the fields into the data property?

Reply:
yeah same question

Reply:
Yeah good point. I think we could actually just include the signed response. Updating. Will cut to staging again & test

Reply:
Updated this test & all of them to use the nested .data field

Reply:
The signed response was still included